hermes/probe/delete/delete_test.go

```diff
@@ -114,13 +114,14 @@ func genTestTarget(cfg *monitorpb.HermesProbeDef, t *testing.T) *probe.Target {
 	}
 }
 
-// setupFakeSystem sets up the fake storage system through the fake client.
-func setupFakeSystem(ctx context.Context, t *testing.T) stiface.Client {
+// fakeStorageClient sets up the fake storage system through the fake client
+// and returns the fake client.
+func fakeStorageClient(ctx context.Context, t *testing.T) stiface.Client {
 	client := mock.NewFakeClient()
 
-	mockBucket := client.Bucket(bucketName)
-	if err := mockBucket.Create(ctx, "", nil); err != nil {
-		t.Fatalf("failed to create mock bucket, err: %v", err)
+	fakeBucket := client.Bucket(bucketName)
```

Comment:
Just `bucket`. The reader already knows everything is a fake here.

Reply:
Ack.